lib/stripe/stripe_client.rb

```diff
@@ -412,7 +428,7 @@ def self.maybe_gc_connection_managers
       last_used_threshold =
         Util.monotonic_time - CONNECTION_MANAGER_GC_LAST_USED_EXPIRY
 
-      pruned_thread_contexts = []
+      pruned_contexts = []
```

Comment:
Ran into lint line length problems on the code above so I gave the equivalent variable a shorter name. I renamed this one too for consistency.